### PR TITLE
Add hardware identification, Ublox 115200 baud and toggle LED when transmitting

### PR DESCRIPTION
**Pull request by l-jost**

### CODE REVIEW

**Review by jbinz — APPROVED** [review]

File: Core/Src/Transmission/Transmission.cpp [hunk 1]
```diff
@@ -183,14 +183,16 @@ void Transmission::disableTransmission() {
   HAL_GPIO_WritePin(LED_GPIO_Port, LED_Pin, GPIO_PIN_SET);
 }
 
-void Transmission::processRFPacket() {
+bool Transmission::processRFPacket() {
```
Comment (stojadin2701) [inline Core/Src/Transmission/Transmission.cpp:191]: document whether what is success -- true or false

File: Core/Src/gps.cpp [hunk 2]
```diff
@@ -19,9 +33,17 @@ void gpsSetup() {
...
-  HAL_UART_Transmit(&huart1, command, 14, 100);
+  // Check hardware version
+  if (HAL_GPIO_ReadPin(HARDWARE_ID_GPIO_Port, HARDWARE_ID_Pin)) {
+    // Flight computer
+    HAL_UART_Transmit(&huart1, ublox_request_115200_baud,
+                      sizeof(ublox_request_115200_baud), 100);
+  } else {
+    // Groundstation
+    /* Request UART speed of 115200 */
+    sprintf((char *)command, "$PCAS01,5*19\r\n");
```
Comment (stojadin2701) [inline Core/Src/gps.cpp]: snprintf

File: Core/Src/gps.cpp [hunk 3]
```diff
@@ -32,16 +54,19 @@ void gpsSetup() {
...
+    /* Request 5 Hz mode */
+    HAL_UART_Transmit(&huart1, ublox_request_5Hz, sizeof(ublox_request_5Hz),
+                      100);
+    /* Request airbourne, not working yet */
+    // HAL_UART_Transmit(&huart1, ublox_request_airbourne,
+    // sizeof(ublox_request_airbourne), 100);
+  } else {
+    // Groundstation
+    /* Request 10Hz update rate */
+    sprintf((char *)command, "$PCAS02,100*1E\r\n");
```
Comment (stojadin2701) [inline Core/Src/gps.cpp]: snprintf

File: Core/Src/main.cpp [hunk 4]
```diff
@@ -257,7 +258,7 @@ int main(void) {
       HAL_UART_Transmit(&huart2, uartOutBuffer, 7, 2);
     }
 
-    if(send_version_num){
+    if (send_version_num) {
```
Comment (stojadin2701) [inline Core/Src/main.cpp:261]: rename to version_num_sent?

**Pull request merged**